osu.Game.Rulesets.Taiko/Edit/Blueprints/HitPlacementBlueprint.cs

```diff
@@ -27,6 +27,12 @@ public HitPlacementBlueprint()
             };
         }
 
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+            BeginPlacement();
```

Comment:
Should not be applied only to placing hits, but also to swells and drum rolls. Compare: osu! slider placement flow, [`SliderPlacementBlueprint`](https://github.com/ppy/osu/blob/master/osu.Game.Rulesets.Osu/Edit/Blueprints/Sliders/SliderPlacementBlueprint.cs).

Reply:
how about put this logic to [PlacementBlueprint](https://github.com/ppy/osu/blob/master/osu.Game/Rulesets/Edit/PlacementBlueprint.cs)?

Seems to be fine and works in all rulesets
just remove the redundant override

Reply:
Maybe? I'm not sure that calling `BeginPlacement()` in BDL will work for placement of every object type ever. I'm not against it, but it could potentially break custom rulesets.

And if we go down this way, then all local `BeginPlacement()` calls that are in code now must be trimmed.

Reply:
Please fix just taiko for now as per updated PR title. We can revisit the global change later.